packages/account/src/Components/api-token/api-token-clipboard.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIsMounted } from '@deriv/shared';
+import { Button, Icon, Modal, Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const WarningNoteBullet = ({ message }) => (
+    <div className='da-api-token__bullet-wrapper'>
+        <div className='da-api-token__bullet' />
+        <Text as='p' color='prominent ' size='xs' line_height='m'>
+            {message}
+        </Text>
+    </div>
+);
+
+const WarningDialogMessage = () => (
+    <>
+        <Text as='p' color='prominent ' size='xs' line_height='m'>
+            {localize(
+                'Be careful who you share this token with. Anyone with this token can perform the following actions on your account behalf'
+            )}
+        </Text>
+        <div className='da-api-token__bullet-container'>
+            <WarningNoteBullet message={localize('Add accounts')} />
+            <WarningNoteBullet message={localize('Create or delete API tokens for trading and withdrawals')} />
+            <WarningNoteBullet message={localize('Modify account settings')} />
+        </div>
+    </>
+);
+
+const ApiTokenClipboard = ({ scopes, text_copy, info_message, success_message, popoverAlignment = 'bottom' }) => {
```

Comment:
`popoverAlignment` needs to be in snake case since it is just a variable name.

Reply:
done :ok_hand: